notebooks/book2/Fig16_1_(book2)_mlpPriorsDemo.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        %pip install --upgrade -q pip jax jaxlib
</pre><p><code>%pip install -qq jax jaxlib</code> should be fine for consistency.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>